### PR TITLE
Added Responsive Blog section

### PR DESCRIPTION
**Pull request by ivelinmegdanov**
### Computer x1920px.
image

### Tablet x768px.
image
image

### Mobile x320px.
image
image
image

### CODE REVIEW

File: styles/main/style.css [hunk 1]
```diff
@@ -403,6 +410,72 @@ html {
...
+}
+.blog__image {
+  width: 100%;
+}
+.blog__date {
+  position: absolute;
+  top: 20px;
+  right: 20px;
+  font-size: 14px;
+}
```
Comment (PetromirDev) [inline styles/main/style.css]: font-weight should be 300

File: index.html [hunk 2]
```diff
@@ -331,6 +331,133 @@ <h2 class="contacts__title">Just Say Hello!</h2>
...
+                        <div class="blog__content">
+                            <div class="blog__interactions">
+                                <p class="interaction__text">25 comments</p>
+                                <p class="interaction__text">17 Likes</p>
+                                <p class="interaction__text">By Adams</p>
+                            </div>
+                            <h2 class="blog__title">Design is not just color it speaks visually...</h2>
+                            <div class="blog__more">
+                                <p class="blog__readmore">Read more</p>
+                                <img class="blog__readmore" src="images/Skills/icons/arrow-white.svg" alt="Arrow"/>
```
Comment (PetromirDev) [inline index.html]: the arrow should be smaller

File: index.html [hunk 3]
```diff
@@ -331,6 +331,133 @@ <h2 class="contacts__title">Just Say Hello!</h2>
...
+                                <img class="blog__image" src="images/Blog/Desktop/shoe.svg" alt="Blog Image">
+                            </picture>
+                            <p class="blog__date">25 July 2020</p>
+                        </div>
+                        <div class="blog__content">
+                            <div class="blog__interactions">
+                                <p class="interaction__text">25 comments</p>
+                                <p class="interaction__text">17 Likes</p>
+                                <p class="interaction__text">By Adams</p>
+                            </div>
```
Comment (PetromirDev) [inline index.html]: Are they interactions tho? I think it's just info about the blog post.

File: styles/main/style.css [hunk 4]
```diff
@@ -403,6 +410,72 @@ html {
...
 
+/* Blogs section */
+.blogs {
+  margin-top: 140px;
+}
+.blogs__title {
+  text-align: center;
+  font-size: 54px;
+  font-weight: 500;
+}
```
Comment (PetromirDev) [inline styles/main/style.css]: I'm pretty sure that blog shouldn't be plural. It's "blog" and the items inside should be "article" or "blog__article". Maybe even- the container - "articles" and each item "article".

**Review by PetromirDev — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (ivelinmegdanov) [reply index.html]: The arrow is the same size for me in both my project and Figma.

**Pull request merged**